runtime-tools-quarkus-extension-parent/runtime/src/main/java/org/kie/kogito/runtime/tools/quarkus/extension/runtime/forms/model/Form.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.kogito.runtime.tools.quarkus.extension.runtime.forms.model;
+
+import io.vertx.core.json.JsonObject;
+
+public class Form {
+
+    public static String SOURCE_KEY = "source-content";
+
+    private String name;
+    private JsonObject source = new JsonObject();
```

Comment:
Source must be String, it should be the raw content of the form file file (html/tsx),